packages/gatsby/src/redux/save-state.ts

```diff
@@ -1,17 +1,16 @@
-const _ = require(`lodash`)
-const report = require(`gatsby-cli/lib/reporter`)
-const { captureEvent } = require(`gatsby-telemetry`)
-const redux = require(`./`)
+import report from "gatsby-cli/lib/reporter"
+import { captureEvent } from "gatsby-telemetry"
+import { saveState as reduxSaveState } from "./"
 
 let saveInProgress = false
-async function saveState() {
+export function saveState(): void {
```

Comment:
Not sure if this should be an arrow function or not? Seems like there's a mixture of uses?